clusters/ithc/base/kustomization.yaml

```diff
@@ -8,6 +8,7 @@ resources:
   - ../../../apps/monitoring/base/kustomize.yaml
   - ../../../apps/neuvector/base/kustomize.yaml
   - ../../../apps/adoption/base/kustomize.yaml
+  - ../../../apps/idam/migration-base/kustomize.yaml
```

Comment:
as above - what is the requirement for `migration-base`?

Reply:
I want to migrate one app in ITHC, without affecting the current deployed state of apps in Preview and Sandbox environments, so I can observe the switchover process as to inform our eventual production release. (speed/downtime)

Reply:
As discussed, temporary - will be switched to` /base/` after reviewing